classificators/src/main/java/ru/okpdmarket/service/SearchService.java

```diff
@@ -11,7 +11,7 @@
  */
 @Service
 public interface SearchService {
-    List<ClassificatorItem> search(String classificatorId, String query);
+    List<ClassificatorItem> findClassificator(String classificatorId, String query);
```

Comment:
Who and why renamed this?(Maybe me?)
we don't 'find classificator' here, but rather  'search in it'